beacon_node/lighthouse_network/src/peer_manager/peerdb.rs

```diff
@@ -169,6 +169,11 @@ impl<E: EthSpec> PeerDB<E> {
         }
     }
 
+    /// Returns the sync start time of the peer if it exists
+    pub fn sync_start_time(&self, peer_id: &PeerId) -> Option<&Instant> {
+        self.peers.get(peer_id).and_then(|info| info.sync_start_time())
```

Comment:
I don't follow the meaning/purpose of sync start time, could you detail it?

Reply:
I added this to calculate sync time per client. When a sync request is received I update this parameter to record the time when the sync started [here](https://github.com/iri031/lighthouse/blob/metrics-by-client/beacon_node/network/src/sync/manager.rs#L417) and later when the state of the peer changes to is_synced [here](https://github.com/iri031/lighthouse/blob/metrics-by-client/beacon_node/network/src/sync/manager.rs#L593)  I calculate the time taken to include in the metric.